.github/workflows/release-standalone.yml

```
@@ -0,0 +1,89 @@
+        run: |
+          cd build/dist
+          zip -qr9 ../../tetra-legends-ultimate-standalone.zip tetra-legends-ultimate
+          cd ../..
+          echo "size=$(du -h tetra-legends-ultimate-standalone.zip | cut -f1)" >> "$GITHUB_OUTPUT"
+
+      - name: Publish release
+        env:
+          GH_TOKEN: ${{ github.token }}
+          TAG: ${{ inputs.tag || github.ref_name }}
+          PRERELEASE: ${{ inputs.prerelease && '--prerelease' || '' }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:27:3: Double quote to prevent globbing and word splitting

(shellcheck)